indico/modules/users/forms.py

```diff
@@ -70,6 +70,16 @@ class UserPreferencesForm(IndicoForm):
         widget=SwitchWidget(),
         description=_('The previewer is used by default for image and text files, but not for PDF files.'))
 
+    add_ical_alerts = BooleanField(
+        _('Add alerts to iCal'),
+        widget=SwitchWidget(),
+        description=_('Add an event reminder to exported iCal files/URLs.'))
+
+    add_ical_alerts_mins = IntegerField(
+        _('Timeout for iCal alerts'),
```

Comment:
Not sure if "Timeout" is the best word here... Can't think of a good alternative though, "delay" doesn't really fit either...